src/pymor/reductors/coercive.py

```diff
@@ -48,9 +49,34 @@ def __init__(self, fom, RB=None, product=None, coercivity_estimator=None,
 
     def assemble_error_estimator(self):
         residual = self.residual_reductor.reduce()
-        error_estimator = CoerciveRBEstimator(residual, tuple(self.residual_reductor.residual_range_dims),
-                                              self.coercivity_estimator)
-        return error_estimator
+
+        # optional output estimate
+        if self.assemble_output_error_estimate and self.fom.output_functional.linear:
+            output_estimator_matrices, output_functional_coeffs = self.assemble_output_error_estimator()
+        else:
+            output_estimator_matrices = output_functional_coeffs = None
+
+        return CoerciveRBEstimator(residual, tuple(self.residual_reductor.residual_range_dims),
+                                   self.coercivity_estimator, output_estimator_matrices, output_functional_coeffs)
+
+    def assemble_output_error_estimator(self):
+        output_estimator_matrices = output_functional_coeffs = []
```

Comment:
This is most likely not what you want! This introduces `output_estimator_matrices` as a an additional name for `output_functional_coeffs`, i.e. the two point to the same list! In this case, `output_functional_coeffs` is overridden below, so it does not matter. Bu I would suggest something like
```suggestion
        output_estimator_matrices, output_functional_coeffs = [], []
```

Reply:
ah yes sure !